cmake/config/config_armhf-linux.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 Samsung Electronics Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(CMakeForceCompiler)
+
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR armhf)
+
+EXECUTE_PROCESS( COMMAND uname -m COMMAND tr -d '\n' OUTPUT_VARIABLE ARCHITECTURE )
+
+if (${ARCHITECTURE} STREQUAL "x86_64" OR  ${ARCHITECTURE} STREQUAL "i686")
+  set(CMAKE_C_COMPILER arm-linux-gnueabihf-gcc)
+endif()
+
+if (${ARCHITECTURE} STREQUAL "arm???")
```

Comment:
When I mention above code segment in #61 , I did not test this code segment on RPI2, thus I said it is not complete code. Could you change `arm???` to the string that your RPI2 returns?